apps/portal/src/app/features/file-processor/file-processor.component.ts

```diff
@@ -13,9 +14,26 @@ export class FileProcessorComponent implements OnInit {
   // to do:the name will probably be changed to match the backend
   jobs: JobDetails[] = [];
 
-  constructor(private http: HttpClient) {}
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  summaries: any[] = [];
+
+  constructor(
+    private http: HttpClient,
+    private fileService: FileService,
+  ) {}
 
   ngOnInit(): void {
+    const jobIds = [1, 2]; // Once file upload api is integrated will change this code
+    this.fileService.fetchSummaries(jobIds).subscribe(
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      (result: any) => {
+        this.summaries = result.data.summaries.summaries;
```

Comment:
What are you showing in table if there is no summaries?

Reply:
currently in grid we are showing data with hardcoded values from api response we are getting job status as '1' what does that mean?
There are 4 status in the enum file

What data do I need to show from API response, response screenshot is attached below
<img width="1190" alt="Screenshot 2024-05-12 at 6 23 34 PM" src="https://github.com/OsmosysSoftware/osm-transcript-summarizer/assets/168435416/d7860b2d-4840-417c-ae2d-ba77919c8550">

Reply:
@Hitesh-Kandpal 
Check values from api folder. And map them

Reply:
mapped the values, also removed the dummy data now we are directly taking the job data from api response and using that we are showing values in UI, look of UI is same as the previous video that I attached in my last comment

Reply:
if there are no summaries(no file uploaded in database) then we are only showing the table columns heading,

Timestamp,File Name,Status,Actions

Reply:
That should not be the case.
You should show a message saying no records or something similar.
In primeNG we have option for showing it through the tempalte pTemplate="emptymessage".
https://primeng.org/table

Also make sure you are sorting the fetch with latest one at the top.

Reply:
<img width="1440" alt="Screenshot 2024-05-13 at 11 37 08 AM" src="https://github.com/OsmosysSoftware/osm-transcript-summarizer/assets/168435416/f80b7516-537f-426d-b804-ab4f23537c6d">

fixed that, also changed the code for whole table in html and used primeng properly

and we are sorting them in ascending order , which means the latest job id will be on top